projects/packages/sync/src/modules/class-comments.php

```diff
@@ -515,4 +521,73 @@ public function expand_comment_ids( $args ) {
 			$previous_interval_end,
 		);
 	}
+
+	/**
+	 * Expand the comment IDs to comment objects and meta before being serialized and sent to the server.
+	 *
+	 * @access public
+	 *
+	 * @param array $args The hook parameters.
+	 * @return array The expanded hook parameters.
+	 */
+	public function extract_comments_and_meta( $args ) {
+		list( $filtered_comments, $previous_end ) = $args;
+		return array(
+			$filtered_comments['objects'],
+			$filtered_comments['meta'],
+			$previous_end,
+		);
+	}
+
+	/**
+	 * Given the Module Configuration and Status return the next chunk of items to send.
+	 * This function also expands the posts and metadata and filters them based on the maximum size constraints.
+	 *
+	 * @param array $config This module Full Sync configuration.
+	 * @param array $status This module Full Sync status.
+	 * @param int   $chunk_size Chunk size.
+	 *
+	 * @return array
+	 */
+	public function get_next_chunk( $config, $status, $chunk_size ) {
+
+		$comment_ids = parent::get_next_chunk( $config, $status, $chunk_size );
+		if ( empty( $comment_ids ) ) {
+			return array();
+		}
+		$comments = get_comments(
+			array(
+				'include_unapproved' => true,
```

Comment:
I know we aim for consistency here but `include_unapproved` is [supposed to be an array of user IDs or email addresses](https://developer.wordpress.org/reference/classes/WP_Comment_Query/__construct/#parameters). Could we fix this or create an issue to audit this in a follow up PR?

Reply:
Yes, looks like this might have been inherited from older usages https://developer.wordpress.org/reference/functions/get_commentdata/.
So we can remove it due to  `status` being all by default in the `WP_Comment_Query` constructor.

Done in https://github.com/Automattic/jetpack/pull/41183/commits/0d50e0507d275a5ee0bbb2772bd6eaedc60f8be8